lib/combo.coffee

```diff
@@ -76,6 +76,10 @@
 
     # show classname if not false or null
     classNameOnEmpty: false
+    

```

Comment:
use to specify which items are rendered with which labels
({ rawValue: string, item: T}) => { text: string, className: string} | null